.repo-metadata.json

```diff
@@ -2,12 +2,13 @@
   "name": "videointelligence",
```

Comment:
does this need to change too?

Reply:
This isn't used in a meaningful way in our templates, so let's leave it as it is.